### PR TITLE
Updates Auth ChangeLog

### PR DESCRIPTION
**Pull request by protocol86**
Updates the Auth ChangeLog for next release.

### CODE REVIEW

File: Firebase/Auth/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# v4.4.1
+- Fixes bug where the FIRAuthResult object returned following a Phone Number authentication
+  always contained a nil FIRAdditionalUserInfo object.  Now the FIRAdditionalUserInfo is never
```
Comment (paulb777) [inline Firebase/Auth/CHANGELOG.md]: is never what?

File: Firebase/Auth/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+  and it's newUser field is populated correctly.
```
Comment (paulb777) [inline Firebase/Auth/CHANGELOG.md]: "its" not "it's"

File: Firebase/Auth/CHANGELOG.md (hunk 1 above)
Comment (protocol86) [reply Firebase/Auth/CHANGELOG.md]: "nil", added.

File: Firebase/Auth/CHANGELOG.md (hunk 2 above)
Comment (protocol86) [reply Firebase/Auth/CHANGELOG.md]: thanks, fixed.

**Pull request merged**